week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -20,21 +20,41 @@ You have to implement the missing features according to the specification.
 // Here is your starting point:
 let bottle = {
   volume: 0,
-  fillUp: function() {
-    // calling this function should pour your bottle full (volume = 100);
+  fillUp: function () {
+    return this.volume === 100;
   },
+
   pour: function () {
     // calling this function should increase your bottle volume by 10 unit;
+
+    if (this.volume <= 90) {
+      this.volume += 10;
+    } else {
+      this.volume === 100;
+    }

```

Comment:
Again, use the single '=' assignment operator, not '==='.

Reply:
done